changelog/@unreleased/pr-4674.v2.yml

```
@@ -0,0 +1,6 @@
+type: improvement
+improvement:
+  description: Disruptor Autobatchers now has a special future that allows it track
+    the different phases of the batching process.
```

nit: I'd word this a bit more directly towards how users will consume the tracking.

```suggestion
  description: Disruptor Autobatchers now trace their calls. Time spent waiting for the
    batch to start and for the batch to be processed are also tracked in child spans.
```